decentraland/components/presentational/Button.js

```diff
@@ -8,34 +8,56 @@ import {
   responsiveFontSize,
 } from "react-native-responsive-dimensions";
 
+// TODO: Change to NativeBase button component
+// https://github.com/tasitlabs/tasit/issues/204
 export default function Button(props) {
+  let { title, onPress, disabled } = props;
+  if (!disabled) disabled = false;
+  if (!onPress) onPress = () => {};
+  title = title.toUpperCase();
+
   return (
     <RNButton
-      containerStyle={styles.container}
-      style={styles.button}
-      onPress={props.onPress}
+      containerStyle={
+        disabled ? disabledStyles.container : enabledStyles.container
+      }
+      style={disabled ? disabledStyles.button : enabledStyles.button}
+      onPress={onPress}
       activeOpacity={1}
     >
-      {props.title.toUpperCase()}
+      {title}
     </RNButton>
   );
 }
 
 Button.propTypes = {
   title: PropTypes.string.isRequired,
-  onPress: PropTypes.func.isRequired,
+  onPress: PropTypes.func,
+  disabled: PropTypes.bool,
 };
 
-const styles = StyleSheet.create({
+const enabledStyles = StyleSheet.create({
   button: {
     fontSize: responsiveFontSize(1.8),
     color: Colors.buttonColor,
   },
   container: {
     padding: responsiveHeight(1.5),
-
     overflow: "hidden",
     borderRadius: 3,
     backgroundColor: Colors.buttonBackground,
   },
 });
+
+const disabledStyles = StyleSheet.create({
+  button: {
+    fontSize: responsiveFontSize(1.8),
```

Comment:
Since 1.8 is used twice, make it a `const` above?

Reply:
Yes, done.

Reply:
Thanks! Resolving